src/storyMap/components/StoryMapUpdate.js

```diff
@@ -64,14 +64,22 @@ const StoryMapUpdate = props => {
 
     const { title, slug, storyMapId, published } = saved;
     setSaved(null);
+    const url = generateStoryMapUrl({ slug, storyMapId });
+
+    console.log({ published, storyMap });
+    const event = published
+      ? storyMap.isPublished
+        ? 'storymap.save'
```

Comment:
question: would update or revise make more sense?
```suggestion
        ? 'storymap.update'
```

Reply:
@paulschreiber I will tag you in the slack thread so that you can coordiante this with Derek